automagicallyexecute/src/executeAutomagically.ts

```diff
@@ -0,0 +1,164 @@
+import {setTimeout} from 'node:timers'
+import {fetchJson} from './fetchJson.js'
+import {ExecuteResponse, TestReport} from './types.js'
+import {
+  debug,
+  getBoolInput,
+  getEndpointAuthorizationParameter,
+  getInput,
+  getInputRequired,
+  getVariable,
+  setResult,
+  TaskResult,
+  warning
+} from 'azure-pipelines-task-lib'
+
+const TIME_BETWEEN_POLLS_MILLISECONDS = 5_000
+const MAXIMUM_POLL_TIME_MILLISECONDS = 2 * 60 * 60 * 1000
```

Comment:
we could mention the maximum poll time of two hours in the readme

Reply:
done